test/Microsoft.AspNetCore.SignalR.Client.FunctionalTests/HubConnectionTests.cs

```diff
@@ -166,6 +166,33 @@ public HubConnectionTests(ServerFixture<Startup> serverFixture, ITestOutputHelpe
 
         [Theory]
         [MemberData(nameof(HubProtocolsAndTransportsAndHubPaths))]
+        public async Task InvokeNonExistantClientMethodFromServer(IHubProtocol protocol, TransportType transportType, string path)
+        {
+            using (StartLog(out var loggerFactory))
+            {
+                var httpConnection = new HttpConnection(new Uri(_serverFixture.BaseUrl + path), transportType, loggerFactory);
+                var connection = new HubConnection(httpConnection, protocol, loggerFactory);
+                try
+                {
+                    await connection.StartAsync().OrTimeout();
+
+                    await connection.InvokeAsync("CallHandlerThatDoesntExist").OrTimeout();
+
```

Comment:
nit: add empty line
I think you may also want to check that there is no exception in the close event. I think if the connection was closed due to an exception it would show up there.